pkg/trace/agent/run.go

```diff
@@ -66,7 +66,7 @@ func Run(ctx context.Context) {
 	if flags.CPUProfile != "" {
 		f, err := os.Create(flags.CPUProfile)
 		if err != nil {
-			log.Critical(err)
+			log.Error(err)
```

Comment:
Why did you change this? If the flag is on for CPU profiling, this is surely critical isn't it?

Reply:
`Critical` errors should force the shutdown of the agent (as per the RFC)